plugins/gestures.koplugin/main.lua

```diff
@@ -1216,7 +1216,8 @@ function Gestures:onFlushSettings()
 end
 
 function Gestures:updateProfiles(action_old_name, action_new_name)
-    for section, gestures in pairs(self.settings_data.data) do -- custom_multiswipes, gesture_fm, gesture_reader
```

Comment:
The old comment seems to imply the way it's written there is necessary for custom multiswipes to function in this context, which is a bit of a separate question from if it crashes or not. ^^

Reply:
Got it, "they" means "custom multiswipes".
Yes, they will still work properly.
By (unexpected) design, the custom_multiswipes section in gestures.lua has different format and doesn't contain actions. All actions for custom multiswipes are stored in fm and reader sections, and will be updated properly when some actions are added to or removed from the dispatcher list.